slides/extras-vetiver.qmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Extras - Model deployment"
+subtitle: "Advanced tidymodels"
+format:
+  revealjs: 
+    slide-number: true
+    footer: <https://workshops.tidymodels.org>
+    include-before-body: header.html
+    include-after-body: footer-annotations.html
+    theme: [default, tidymodels.scss]
+    width: 1280
+    height: 720
+knitr:
+  opts_chunk: 
+    echo: true
+    collapse: true
+    comment: "#>"
+    fig.path: "figures/"
+---
+
+```{r setup}
+#| include: false
+#| file: setup.R
+```
+
+# Deploy your model `r hexes("vetiver")`
+
+## {background-image="https://vetiver.rstudio.com/images/ml_ops_cycle.png" background-size="contain"}
+
+```{r setup-previous}
+#| echo: false
+library(countdown)
+library(tidymodels)
+
+set.seed(123)
+taxi_split <- initial_split(taxi, prop = 0.8, strata = tip)
+taxi_train <- training(taxi_split)
+taxi_test <- testing(taxi_split)
+
+tree_spec <- decision_tree(cost_complexity = 0.0001, mode = "classification")
+tree_fit <- workflow(tip ~ ., tree_spec) %>% fit(taxi_train)
+```
+
+## Deploying a model `r hexes("vetiver")`
```

Comment:
Do we need to talk a little bit about where `tree_fit` and `taxi` comes from since this is technically an Advanced deck?

Just a sentence i fine